packages/core/package.json

```diff
@@ -70,8 +70,7 @@
     "@sentry/cli": "2.39.1",
     "@sentry/core": "8.41.0",
     "@sentry/react": "8.41.0",
-    "@sentry/types": "8.41.0",
-    "@sentry/utils": "8.41.0"
```

Comment:
Despite being deprecated, I'd keep it for now so that it doesn't generate a break change to users that are using @sentry/utils

Reply:
Good catch @lucas-zimerman 🙇 
Reverted with 44a2b3f2edab144a0a8ee773a657f5956fa9da9e